vibes/vibed-ui/src/ui/Theme.cpp

```
@@ -0,0 +1,196 @@
+        ++numberEnd;
+    }
+    if (numberEnd <= numberBegin) {
+        return false;
+    }
+
+    const std::string numberText = json.substr(numberBegin, numberEnd - numberBegin);
+    value = std::atoi(numberText.c_str());
+    return true;
+}
+
+int hexDigitToInt(char c) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)